pushtemplates/src/main/java/com/clevertap/pushtemplates/PushTemplateReceiver.java

```diff
@@ -415,7 +415,7 @@ private void handleRatingNotification(Context context, Bundle extras) {
             int notificationId = extras.getInt(Constants.PT_NOTIF_ID);
             if (extras.getBoolean("default_dl", false)) {
                 notificationManager.cancel(notificationId);
-
+                Utils.raiseNotificationClicked(context,extras);
```

Comment:
@vagnihotri We could add this here or maybe get this launch intent to call `PTNotificationIntentService` What say?